### PR TITLE
Issue #355: Enable Views module on D7 upgrade.

### PR DESCRIPTION
**Pull request by ghost**
Addresses https://github.com/backdrop/backdrop-issues/issues/355.

Not sure if I did the config stuff right, particularly the `field_rewrite_elements` object...

### CODE REVIEW

File: core/modules/system/system.install [hunk 1]
```diff
@@ -2109,6 +2109,52 @@ function system_update_1024() {
...
+  // Install default config.
+  $config = config('views.settings');
+  $config->set('no_javascript', FALSE);
+  $config->set('exposed_filter_any_label', FALSE);
+  $config->set('show_additional_queries', FALSE);
+  $config->set('sql_signature', FALSE);
+  $config->set('devel_output', FALSE);
+  $config->set('devel_region', 'footer');
+  $config->set('display_extenders', array());
+  $config->set('field_rewrite_elements', (object) array(
```
Comment (quicksketch) [inline core/modules/system/system.install]: No need for the (object) casting here.

**Comment by quicksketch** [comment]
Very nice! It looks very close. And although it probably does not hurt, we don't need to cast `field_rewrite_elements` to an object. When it gets converted to JSON, all associative arrays are converted to objects anyway (since JSON does not have support for associative arrays).

**Comment by ghost** [comment]
Cool!

**Comment by jenlampton** [comment]
looks like this one just needs a rebase?

**Comment by ghost** [comment]
Already fixed: https://github.com/backdrop/backdrop/pull/555

**Pull request closed without merging**